src/coq_elpi_glob_quotation.ml

```diff
@@ -182,7 +182,11 @@ let rec gterm2lp ~depth state x =
 
   | GEvar(_k,_subst) -> nYI "(glob)HOAS for GEvar"
   | GPatVar _ -> nYI "(glob)HOAS for GPatVar"
-(*   | GProj _ -> nYI "(glob)HOAS for GProj" *)
+  | GProj ((ref,us),args,c) ->
```

Comment:
Just to be sure: is this node just about the projection syntax? (because I now have a way to represent primitive projections)

Reply:
As of coq/coq#14598, `GProj` strictly corresponds to the `t.(f args)` syntax. However, it is not clear to me what will then be the consensus about the role to give to the `GProj`, whether it matches `CProj` (i.e. the `t.(f args)` syntax) or the `Proj` kernel syntax (i.e. the "primitive" projections), or if `t.(f args)` and kernel-primitive get eventually identified.

Reply:
So, I don't know what it implies for elpi. I guess it depends if the elpi way to represent primitive projections is about kernel `Proj` or about `t.(f args)`.

Reply:
> For instance, in [coq/coq#14598](https://github.com/coq/coq/pull/14598), pretyping map `GProj` to `Proj` when the projection is registered as primitive and to `App` if not.

ok, then I'll try to push a commit on top of yours

Reply:
ok

Reply:
Sorry, I need https://github.com/LPCIC/coq-elpi/pull/256 to go in first, then I'll rebase this one and add a few lines to it